crates/forge/tests/e2e/packages_validation.rs

```diff
@@ -0,0 +1,149 @@
+use std::{fs, process::Stdio};
+
+use super::common::runner::runner;
+use assert_fs::prelude::PathChild;
+use camino::Utf8PathBuf;
+use clap::ValueEnum;
+use forge::Template;
+use project_root::get_project_root;
```

Comment:
This package is just a single function with a couple of lines, maybe we could just implement it ourselves?

Reply:
We can but don't wan to mix changes in this PR, can be addressed in separate one.